js/localization/number.js

```diff
@@ -235,6 +235,21 @@ const numberLocalization = dependencyInjector({
         });
     },
 
+    getNegativeEtalonRegExp: function(format) {
```

Comment:
Could you add tests for [localization/ldml](https://github.com/DevExpress/DevExtreme/blob/20_2/testing/tests/DevExpress.localization/ldml.tests.js#L264)?

Reply:
added